### PR TITLE
[4.1][bug] Restore image

### PR DESCRIPTION
**Pull request by dgrammatiko**
Pull Request for Issue #37448 .

### Summary of Changes

Use the right variable

### Testing Instructions

Check that finder has images

### Documentation Changes Required

No

### CODE REVIEW

**Comment by richard67** [comment]
I have tested this item :red_circle: unsuccessfully on 3abe8fd2a0557db6fbaa8db938301b73e510385b<br /><br />Images are included in search results when this PR is applied, but the "Result Image" option of the Smart Search component is not respected, i.e. the images are still included when this option has been switched off and the index has been cleared and rebuilt after that.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37450">issues.joomla.org/tracker/joomla-cms/37450</a>.</sub>

**Comment by richard67** [comment]
Not sure with my test result since this PR fixes that images were not shown at all, and fixing the other thing (option not respected) is another issue and should be fixed separately. But maybe we can fix it here?

**Comment by richard67** [comment]
I think I found it. It was also broken by PR #37022 . Before that PR, variable `$image` was only set when `$showImage` was true, so it did not need to check that again in the `if` condition fixed by this PR. But after the other PR it would need that check. I will make a suggestion in a minute.

File: components/com_finder/tmpl/search/default_result.php [hunk 1]
```diff
@@ -72,9 +72,9 @@
 }
 ?>
 <li class="result__item">
-	<?php if (isset($image)) : ?>
+	<?php if (isset($this->result->imageUrl)) : ?>
```
Comment (richard67) [inline components/com_finder/tmpl/search/default_result.php]:
```suggestion
	<?php if ($showImage && isset($this->result->imageUrl)) : ?>
```
This should fix the issue that images are also shown when the option to show them is switched off.

**Comment by brianteeman** [comment]
> I think I found it. It was also broken by PR https://github.com/joomla/joomla-cms/pull/37022

Exactly what I reported

**Comment by richard67** [comment]
> > I think I found it. It was also broken by PR #37022
> 
> Exactly what I reported

@brianteeman Where? I saw your report handles the images never being shown. My comment refers to the option being ignored and the images being always shown when the first issue is fixed.

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on c1f98b06d4135f19a5e5086e174046dece5c0ae9<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37450">issues.joomla.org/tracker/joomla-cms/37450</a>.</sub>

**Comment by richard67** [comment]
@brianteeman The PR is fine now. Could you test it, too? Thanks in advance.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on c1f98b06d4135f19a5e5086e174046dece5c0ae9<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37450">issues.joomla.org/tracker/joomla-cms/37450</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37450">issues.joomla.org/tracker/joomla-cms/37450</a>.</sub>

**Comment by laoneo** [comment]
Thanks!

**Comment by woluweb** [comment]
Txs for the fix!

One thing that I was wondering though @dgrammatiko : the plugin Responsive Images works automatically a.o. for intro & full images, for images within the articles, even in Custom HTML Modules.
But why doesn't it trigger for the Smart Search Result Page?

Indeed I get
`L'Ombre de Saint-Saëns`
and not the full figure/picture element like on the Article page itself
```html
<figure class="item-image">
	<picture class="responsive-image"><source type="image/webp" sizes="(max-width: 1920px) 100vw 1920px" srcset="/media/cached-resp-images/images/concerts/bw1_1920.webp?version=22e1d71d7cb6fd0e9b7e14fad9ef7730 1920w, /media/cached-resp-images/images/concerts/bw1_1600.webp?version=22e1d71d7cb6fd0e9b7e14fad9ef7730 1600w, /media/cached-resp-images/images/concerts/bw1_1200.webp?version=22e1d71d7cb6fd0e9b7e14fad9ef7730 1200w, /media/cached-resp-images/images/concerts/bw1_992.webp?version=22e1d71d7cb6fd0e9b7e14fad9ef7730 992w, /media/cached-resp-images/images/concerts/bw1_768.webp?version=22e1d71d7cb6fd0e9b7e14fad9ef7730 768w, /media/cached-resp-images/images/concerts/bw1_480.webp?version=22e1d71d7cb6fd0e9b7e14fad9ef7730 480w, /media/cached-resp-images/images/concerts/bw1_320.webp?version=22e1d71d7cb6fd0e9b7e14fad9ef7730 320w, /media/cached-resp-images/images/concerts/bw1_200.webp?version=22e1d71d7cb6fd0e9b7e14fad9ef7730 200w"><source type="image/jpeg" sizes="(max-width: 1920px) 100vw 1920px" srcset="/media/cached-resp-images/images/concerts/bw1_1920.jpg?version=22e1d71d7cb6fd0e9b7e14fad9ef7730 1920w, /media/cached-resp-images/images/concerts/bw1_1600.jpg?version=22e1d71d7cb6fd0e9b7e14fad9ef7730 1600w, /media/cached-resp-images/images/concerts/bw1_1200.jpg?version=22e1d71d7cb6fd0e9b7e14fad9ef7730 1200w, /media/cached-resp-images/images/concerts/bw1_992.jpg?version=22e1d71d7cb6fd0e9b7e14fad9ef7730 992w, /media/cached-resp-images/images/concerts/bw1_768.jpg?version=22e1d71d7cb6fd0e9b7e14fad9ef7730 768w, /media/cached-resp-images/images/concerts/bw1_480.jpg?version=22e1d71d7cb6fd0e9b7e14fad9ef7730 480w, /media/cached-resp-images/images/concerts/bw1_320.jpg?version=22e1d71d7cb6fd0e9b7e14fad9ef7730 320w, /media/cached-resp-images/images/concerts/bw1_200.jpg?version=22e1d71d7cb6fd0e9b7e14fad9ef7730 200w"><img loading="lazy" width="2000" height="1334" src="/media/cached-resp-images/images/concerts/bw1_1920.jpg?version=22e1d71d7cb6fd0e9b7e14fad9ef7730" itemprop="image" alt="L'Ombre de Saint-Saëns"></picture>			<figcaption class="caption">© Leslie Artamonow</figcaption>
	</figure>
```

**Comment by dgrammatiko** [comment]
@woluweb could you open an issue in the repo?
Thanks

**Comment by woluweb** [comment]
Sure. Here it is: https://github.com/ttc-freebies/plugin-responsive-images/issues/162

**Pull request merged**